local-postgresql-copy/copy-data.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+echo "COPYING DATA FROM CLOUD SPANNER TO POSTGRESQL"
+
+# First truncate all tables.
+PGPASSWORD="$POSTGRES_PASSWORD" psql -v ON_ERROR_STOP=1 -U "$POSTGRES_USER" -h "$POSTGRES_HOST" -p "$POSTGRES_PORT" -d "$POSTGRES_DB" -qAtX \
+  -c "select table_name from information_schema.tables where table_type='BASE TABLE' and table_schema='public';" \
+| while read table_name ; do
+  echo ""
+  echo "Truncating data for $table_name"
+  PGPASSWORD="$POSTGRES_PASSWORD" psql -a -v ON_ERROR_STOP=1 -U "$POSTGRES_USER" -h "$POSTGRES_HOST" -p "$POSTGRES_PORT" -d "$POSTGRES_DB" -c "truncate $table_name cascade"
```

Comment:
Will (or should) this error if the table is missing?  (I forget when psql returns what error codes.)